### ISSUE TITLE
Error semver is not in the npm registry

### ISSUE THREAD

**Issue by coreysmithdesign** [kentcdodds/react-fundamentals#78]
I can't seem to get past the setup script. It keeps giving the same error each time.

I've tried removing and re-cloning the repo. 

I've also tried the following command:

```npm set registry https://registry.npmjs.org/```

My environment is:

```
Node...14.4.0
NPM...6.14.8
Git...git version 2.21.0 (Apple Git-122)
```

The error is:

```
git clone https://github.com/kentcdodds/react-fundamentals.git
cd react-fundamentals
npm run setup --silent
Cloning into 'react-fundamentals'...
remote: Enumerating objects: 33, done.
remote: Counting objects: 100% (33/33), done.
remote: Compressing objects: 100% (33/33), done.
remote: Total 1057 (delta 20), reused 1 (delta 0), pack-reused 1024
Receiving objects: 100% (1057/1057), 1.61 MiB | 464.00 KiB/s, done.
Resolving deltas: 100% (736/736), done.
▶️  Starting workshop setup...
      Running the following command: npx "https://gist.github.com/kentcdodds/bb452ffe53a5caa3600197e1d8005733" -q
npx: installed 1 in 3.284s
    ▶️  Starting: System Validation
          Ensuring the correct versions of tools are installed on this computer.
          Running the following command: npx "https://gist.github.com/kentcdodds/abbc32701f78fa70298d444c2303b6d9"
npm ERR! code E404
npm ERR! 404 Not Found - GET http://registry.npmjs.org/semver
npm ERR! 404 
npm ERR! 404  'semver@7.1.3' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'workshop-computer-validator'
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/corey/.npm/_logs/2020-10-04T05_18_22_294Z-debug.log
Install for [ 'git+https://gist.github.com/abbc32701f78fa70298d444c2303b6d9.git' ] failed with code 1
    🚨  Failure: System Validation. Please review the messages above for information on how to troubleshoot and resolve this issue.
```

**Comment by jmagrippis** [comment]
Hm, it's the first command that has a dependency, so it could have been a problem with the npm registry or your setting there.

Does it still happen, when you run just that gist?

```sh
npx "https://gist.github.com/kentcdodds/abbc32701f78fa70298d444c2303b6d9"
```

**Comment by coreysmithdesign** [comment]
It does still do it with just the gist but I also found I can't use create-react-app anymore, so now I don't think it's this script per se. NPX always returns the error code E404.

```
npx "https://gist.github.com/kentcdodds/abbc32701f78fa70298d444c2303b6d9"
npm ERR! code E404
npm ERR! 404 Not Found - GET http://registry.npmjs.org/semver
npm ERR! 404
npm ERR! 404  'semver@7.1.3' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'workshop-computer-validator'
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/corey/.npm/_logs/2020-10-04T10_04_42_008Z-debug.log
Install for [ 'git+https://gist.github.com/abbc32701f78fa70298d444c2303b6d9.git' ] failed with code 1
```

create-react-app

```
npx create-react-app stuff
npm ERR! code E404
npm ERR! 404 Not Found - GET http://registry.npmjs.org/create-react-app
npm ERR! 404
npm ERR! 404  'create-react-app@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/corey/.npm/_logs/2020-10-04T10_07_06_875Z-debug.log
Install for [ 'create-react-app@latest' ] failed with code 1
```

**Comment by coreysmithdesign** [comment]
Figured it out. Something in my environmont, maybe on the network, is blocking me from reaching `http://registry.npmjs.org` but I can reach `https://registry.npmjs.org`.

My global setting was being overriden by the `.npmrc` in the project directory. I changed it to point to the encrypted registry also and it worked! On the off chance somebody else has a similar issue I put this in the `.npmrc` file:

```
registry=https://registry.npmjs.org/
package-lock=true
```

All tests passed. Thanks! I'm looking forward to the workshops!!
